mj_models/include/mj_models/mj_mushr.hpp

```diff
@@ -20,4 +33,16 @@ inline void copy(Ctrl ctrl_out, const mj_models::MushrControl& msg)
   ctrl_out[1] = msg.velocity.data;
 }
 
+template <typename Sensor>
+inline void state_from_sensors(mj_models::MushrState& msg, const Sensor& sensordata)
```

Comment:
See above -- since these will not be the sensors that we will be using for the real system (or even a simulated version in experiments), we should ideally make the distinction between the ground truth pose and what a sensor provides.